src/NServiceBus.RabbitMQ/Routing/ISupportNativeDelays.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace NServiceBus.Transport.RabbitMQ
+{
+    using global::RabbitMQ.Client;
+
+    /// <summary>
+    /// An extension for <see cref="IRoutingTopology"/> implementations which allows
+    /// for receiving messages from the delay infrastructure.
```

Comment:
I like `ISupportDelayedDelivery` as it explains the intent better. In theory you can implement it however you like.